### PR TITLE
boskos core: don't attempt to create CRDs

### PR DESCRIPTION
**Pull request by ixdy**
We recently had a brief outage because we changed some of the names in the Boskos CRDs, and updating these right now is tricky since Boskos attempts to create CRDs itself.

A simpler solution would be to just put the CRDs in YAMLs that can be applied with the rest of the Kubernetes resources, and not have Boskos attempt to create them itself. I'm working on that in a separate PR, to try to isolate concerns. (The existing code already does nothing if the CRDs already exist.)

Credit: @alvaroaleman already suggested doing this in https://github.com/kubernetes/test-infra/pull/16206#discussion_r377559436.

/assign @alvaroaleman @stevekuznetsov

### CODE REVIEW

**Comment by alvaroaleman** [comment]
It has conflicts with your other PR :grimacing:

File: prow/cluster/boskos.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ metadata:
 rules:
 - apiGroups: ["apiextensions.k8s.io"]
   verbs: ["*"]
-  resources: ["customresourcedefinitions"]
+  resources: ["customresourcedefinitions"]  # TODO(ixdy): remove once Boskos no longer attempts to create CRDs itself
```
Comment (fejta) [inline prow/cluster/boskos.yaml]: Prefer an issue over a user

Comment (ixdy) [reply prow/cluster/boskos.yaml]: Done.

**Pull request merged**